src/core/surface/call.c

```diff
@@ -174,6 +174,9 @@ struct grpc_call {
   /* Received call statuses from various sources */
   received_status status[STATUS_SOURCE_COUNT];
 
+  /* Call stats: only valid after trailing metadata received */
```

Comment:
They will be: we've got some small tweaks to cleanup in #2286 that I want to get in first, and then I'll make them available on the destroy edge.